samples/templates/dicomcast-azuredeploy.json

```diff
@@ -0,0 +1,385 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
how do we keep this in sync with the dicom, fhir and dicom-cast templates? Is there a way to compose them?

Reply:
@smithago  investigating how to do this , thanks!

Reply:
@smithago I talked to Brandon, he recommended we could have the template run as part of a scheduled pipeline to see if it is still working. While I would probably be the one who makes changes to dicom or dicom cast template, I wouldn't catch a fhir change that would break template. 

What do you think of adding it into a scheduled pipeline? Thanks